src/main/java/task/Deadlines.java

```diff
@@ -0,0 +1,24 @@
+package task;
+
+public class Deadlines extends Task {
```

Comment:
As the object of this class is only for one Deadline task, the name of the class should probably be Deadline.